src/components/main-page/Location/Location.tsx

```diff
@@ -0,0 +1,51 @@
+import { Heading, Section, Icon } from "components/ui";
+import useTranslation from "translations/hooks/useTranslations";
+import styles from "./Location.module.scss";
+import { mapMarker } from "components/ui/Icon";
+import dynamic from "next/dynamic";
+
+const LocationMap = dynamic(
+  () =>
+    import("./LocationMap").then(
+      ({ LocationMap }) => LocationMap,
+    ) as Promise<React.FC>,
+  {
+    ssr: false,
+  },
+);
+
+export function Location() {
+  const { t } = useTranslation();
+
+  return (
+    <Section variant="blue" id="location" className={styles.section}>
+      <Heading className={styles.sectionHeading} withAccent>
+        {t("location.title")}
+      </Heading>
+      <div className={styles.wrapper}>
+        <LocationMap />
+        <div className={styles.content}>
+          <div className={styles.contentWrapper}>
+            <div className={styles.iconWrapper}>
+              <Icon className={styles.icon} icon={mapMarker} />
+            </div>
+            <div className={styles.headingWrapper}>
+              <p className={styles.heading}>
+                <span className={styles.bold}>
+                  {t("location.location1")} <br></br>
```

Comment:
`<br>` doesn't need closing tag - same in line 37. 

Alternatively, I'd consider wrapping each line in separate paragraph.

Reply:
Yeah, but i guess there some problem with using `<br>` tag inside react? I'm getting this error when trying to use single `<br> `

```
(property) JSX.IntrinsicElements.br: DetailedHTMLProps<HTMLAttributes<HTMLBRElement>, HTMLBRElement>
JSX element 'br' has no corresponding closing tag.ts(17008)
```

So I guess we can switch to using a different paragraph for each line 😁

Reply:
Cause in React you may need to use self closing tag? I mean `<br/>` instead of `<br>`? 🤔 That weird, but it might be that - ref: https://stackoverflow.com/a/57143792

Reply:
I tried both actaully, with self-closing I got this error 
![image](https://user-images.githubusercontent.com/45982530/189531485-20fda715-fde2-4523-8843-9a400d1dc546.png)